data/rounded-everything/Preset.toml

```diff
@@ -0,0 +1,43 @@
+slug = "rounded"
+name = "Rounded Everything"
+creator = "ERROR-404-NULL-NOT-FOUND"
+description = "Everything is rounded! Colors from Dracula theme by Wild"
```

Comment:
Since you're using custom CSS with class specifiers, this needs an associated commit hash.